service/src/main/java/uk/gov/hmcts/reform/fpl/service/MessageJudgeService.java

```diff
@@ -37,4 +52,31 @@ protected String buildMessageHistory(String message, String history, String send
 
         return join("\n \n", history, formattedLatestMessage);
     }
+
```

Comment:
Not sure if these helpers are needed really. Looks like we're trying to handle the JUDICARY and CTSC cases and then do something different for the other cases?

Wonder if it would be easier to follow if we had methods specific to Judicairy CTSC?

Reply:
yeah, this method looks stupid, but I am trying to avoid copying the if-else checking every time I need to resolve Judge/CTSC email.